services/githubService.js

```diff
@@ -1,29 +1,40 @@
-const { fetch } = require('../utils/fetch')
+const axios = require('axios')
```

Comment:
Why remove `fetch`? 🤔

Reply:
We could use the custom fetch if we were doing something extra in our custom function...
Like we could add the `credentials: include` code in our custom fetch, making the call in the controller/service easier to read.
But here it was a simple get request, reduces the call-stack in case of finding errors

Reply:
Then the whole purpose of wrapper function gets lost. We might not use axios tomorrow. Will you then sit and manually change `axios` to `axios-alternative` in every network call? 🤔 
For stale, open PRs and PRs by users.. we have limited network calls now.. but what if have 100s of them?

Reply:
Cool... I just changed the name in the module scope so that there is no confusion between node's fetch and our fetch wrapper 👍🏾

Reply:
Done 👍🏾